src/java/org/apache/cassandra/db/virtual/SystemViewsKeyspace.java

```diff
@@ -28,6 +31,8 @@
 
 public final class SystemViewsKeyspace extends VirtualKeyspace
 {
+    public static final String STORAGE_OPERATIONS = "storage_operations_history";
```

Comment:
what other operations do you plan to add? now it seems like it is just about "compaction". Should not we reflect that? There is also "thread_pools" not on constant if we play the constants game here.

Reply:
Updated to the `compaction_operations_history`